### PR TITLE
Combine repos

### PR DESCRIPTION
**Pull request by nro2**
The version checking is working.  I just had it publish a dummy version (1.2.4) which I just unpublished afterwards, and it works.  It will always run CI on changes to master, and if the version in package.json differs from what shows up in `npm view [package]` then it runs the publish script.

Let me know if any of you see anything you want me to fix, and I will go ahead and start writing some docs.

### CODE REVIEW

**Comment by minorgal** [comment]
Looks nice and simple. Just need docs and this will be ready to merge.

**Review by minorgal — APPROVED** [review]
Looks good. @bre80 should review.

**Review by jordanaustin — COMMENTED** [review]
I like it. Thanks for working on this. I think we're just waiting for @bre80 to review

**Pull request merged**
